Lib/distutils/msvc9compiler.py

```diff
@@ -672,7 +672,7 @@ def manifest_setup_ldargs(self, output_filename, build_temp, ld_args):
     def manifest_get_embed_info(self, target_desc, ld_args):
         # If a manifest should be embedded, return a tuple of
         # (manifest_filename, resource_id).  Returns None if no manifest
-        # should be embedded.  See http://bugs.python.org/issue7833 for why
+        # should be embedded.  (see http://bugs.python.org/issue7833 for why
         # we want to avoid any manifest for extension modules if we can)
```

Comment:
Change trailing ')' to '.' and leave 'See' capitalized.
```suggestion
        # should be embedded.  See http://bugs.python.org/issue7833 for why
        # we want to avoid any manifest for extension modules if we can.
```